yarn.lock

```diff
@@ -3891,6 +3891,13 @@ browserslist@^4.0.0, browserslist@^4.16.0, browserslist@^4.16.6, browserslist@^4
     node-releases "^2.0.13"
     update-browserslist-db "^1.0.13"
 
+bser@2.1.1:
+  version "2.1.1"
+  resolved "https://registry.npmjs.org/bser/-/bser-2.1.1.tgz#e6787da20ece9d07998533cfd9de6f5c38f4bc05"
```

Comment:
Shouldn't these be from the yarn registry?

Reply:
Weird. Fixed now.